models/Element/Service.php

```diff
@@ -117,6 +117,40 @@ public static function getTypePath($element)
         return $path;
     }
 
+
+    /**
+     * @static
+     * @internal
+     *
+     * @param DataObject|Document $element
+     *
+     * @return string
+     *
+     * @throws \Exception
+     */
+    public static function getSortIndexPath($element)
```

Comment:
Which branch is this based?

Reply:
I have pulled and merged the pimcore/master with our forked master branch. After that I had checkout the branch treenpaging. Or should I create a new pull request based from master?

Reply:
please use the master as a basis

Reply:
Okay, I had done.